prow/cluster/boskos.yaml

```diff
@@ -11,10 +11,6 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-apiVersion: v1
```

Comment:
Why ? is this already part of the BUILD_CLUSTER definition ?

Reply:
This is an invalid yaml file. The following three lines of code overwrite these three.

And yes, a) the test-pods namespace already exists and b) we should not be define it in boskos.yaml